### PR TITLE
Fk-3

### PR DESCRIPTION
**Pull request by FilipKaczmarek**

### CODE REVIEW

**Comment by FilipKaczmarek** [comment]
- Background image added
- HTML Structure Created 
- Home Module JS Created and imported to index.js
- Added content - time and date

File: src/JSmodules/Home.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export class Home {
+    constructor() {
+        this.textElement = document.querySelector('.view__textContent');
+    
+        this.start();
+    };
+
+    start() {
+        setInterval(() => {
+            this.fullDate = new Date();
```
Comment (pdybowski) [inline src/JSmodules/Home.js:10]: I will recomend to put this interval in separate method e.g. _timer() or something like this

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { Home }from './JSmodules/Home.js'
+
+// The following two lines of code are responsible for updating the date and time on the home page
+const homePageUpdate = new Home()
+homePageUpdate.start()
```
Comment (pdybowski) [inline src/index.js:5]: dont you run start function twice? in a class constructor and here

File: src/index.html [hunk 3]
```diff
@@ -18,12 +18,32 @@
...
+        <!-- CSS -->
+        <link rel="stylesheet" href="./style.css" />
         <title></title>
     </head>
 
     <body>
+        <div class="wrapper">
+            <div class="wrapper--transparency"></div>
+            <header></header>
+            <main id="main" class="container-lg">
```
Comment (pdybowski) [inline src/index.html:30]: we are going to inject main content dynamically. Each view will be injected here (probably here) so when user switches to another view your content will be removed and never dislayed again

**Review by pdybowski — CHANGES_REQUESTED** [review]
please fix

**Comment by FilipKaczmarek** [comment]
@pdybowski changes have been made

**Comment by pdybowski** [comment]
please change the UI for the one I asked on discord

**Pull request merged**